sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md

```diff
@@ -20,7 +20,8 @@ irregular-plural-words:
   quota: quota
 
 rename-mapping:
-  LoadTestResource: LoadTesting
+  Resource: LoadTestingBaseResource
+  LoadTestResource: LoadTestingResource
   QuotaResource: LoadTestingQuota
   CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailabilityResponse
```

Comment:
CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailability ?

Reply:
This is a common naming pattern for all CheckAvailability method and related models